test/api-test.js

```diff
@@ -413,9 +413,11 @@ describe('RippleAPI', function() {
   });
 
   it('generateWallet', function() {
-    withDeterministicPRNG(() => {
-      assert.deepEqual(this.api.generateWallet(), responses.generateWallet);
-    });
+    function random() {
+      return _.fill(Array(16), 0);
+    }
+    assert.deepEqual(this.api.generateWallet({randGen: random}),
```

Comment:
Ah, right, that would be nice eh: `{random}`
On 14 Aug 2015 05:24, "Chris Clark" <notifications@github.com> wrote:

> In test/api-test.js
> <https://github.com/ripple/ripple-lib/pull/502#discussion_r37032903>:
>
> > @@ -413,9 +413,11 @@ describe('RippleAPI', function() {
> >    });
> >
> >    it('generateWallet', function() {
> > -    withDeterministicPRNG(() => {
> > -      assert.deepEqual(this.api.generateWallet(), responses.generateWallet);
> > -    });
> > +    function random() {
> > +      return _.fill(Array(16), 0);
> > +    }
> > +    assert.deepEqual(this.api.generateWallet({randGen: random}),
>
> Oh, I meant randGen -> random
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ripple/ripple-lib/pull/502/files#r37032903>.
>